### ISSUE TITLE
weird lag

### ISSUE THREAD

**Issue by brawlofnight** [w-okada/voice-changer#769]
### Issue Type

Bug Report

### vc client version number

MMVCServerSIO_mac_onnxcpu-nocuda_v.1.5.3.12a

### OS

Macbook M2 Chip

### GPU

cpu

### Clear setting

yes

### Sample model

no

### Input chunk num

no

### Wait for a while

The GUI successfully launched.

### read tutorial

yes

### Extract files to a new folder.

no

### Voice Changer type

RVC

### Model type

Anaconda

### Situation

I try to use different voices and settings but the res is always really high, the lowest it gets is 18k and the highest is around 50k. I don't know if it's a problem only for M2. I just downloaded it yesterday and was trying to figure out how to get the UI to show up I saw the tutorial and it told me to download Anaconda if I'm using Mac. I did, and the UI showed up and now I'm trying it; it is super laggy and choppy too. I haven't tried using my voices downloaded from AIHUB but I don't know. In the pictures, you can tell me what settings are best because I'm still new to the platform and all the content posted about w Okada is using Microsoft.

### application window capture

https://imgur.com/a/sufVi1d
https://imgur.com/a/bLzZkHw

### logs on terminal

https://docs.google.com/document/d/11GPUpRANgWDzLNJkxa7g5BEWl-DcipJOysRpGzf_zoo/edit?usp=sharing

**Comment by w-okada** [comment]
use dio, increase chunk, decrease extra, set rvc quality low

**Comment by brawlofnight** [comment]
> use dio, increase chunk, decrease extra, set rvc quality low

whats a dio and how do I set rvc to low quality?

**Comment by brawlofnight** [comment]
Alright, now I changed the settings but the res starts low but keeps increasing Im still at the average of 34k res. Is it because of the S. Thresh or the gpu

**Comment by brawlofnight** [comment]
<img width="86" alt="Screenshot 2023-08-28 at 6 41 11 AM" src="https://github.com/w-okada/voice-changer/assets/74514895/d8627ae4-18c2-41e8-aae8-3d1c30287dc1">

**Comment by brawlofnight** [comment]
and what chunk number should it be? like give the most details you can give thx

**Comment by w-okada** [comment]
share window capture.

**Comment by brawlofnight** [comment]
https://imgur.com/a/prOv2V2
https://imgur.com/a/YKJs3QM
https://imgur.com/a/PlBo2mJ

**Comment by brawlofnight** [comment]
do I need a graphics chip or something? like any tips

**Comment by exciar** [comment]
try a windows machine

**Comment by brawlofnight** [comment]
none of them work/not free could you tell me some free ones that work for m2 chip MacBook air

**Comment by w-okada** [comment]
how about sample model?

**Comment by brawlofnight** [comment]
> 
Yeah ive tried using the sample models and most of it is chunky/it sounds like the person is wheezing and cutting out. it is very hard to use and keeps on making weird sounds

**Comment by w-okada** [comment]
what is your MacOS version ?

**Comment by brawlofnight** [comment]
Apple m2 ventura silicone

**Comment by w-okada** [comment]
I can use this application on Apple M1 Venture. I don't have M2, so I cannot reproduce...

**Comment by brawlofnight** [comment]
> 

I can open it but it is just extremely laggy

**Comment by Luciel007** [comment]
I'm having the exact same issue hoping to find an answer here, has anyone figured out a possible solution? I have windows 11 with a ryzen 7 4800h, 16gb ram and an rtx 2060 there's no reason why I shouldn't be able to run this software properly

**Comment by brawlofnight** [comment]
> I'm having the exact same issue hoping to find an answer here, has anyone figured out a possible solution? I have windows 11 with a ryzen 7 4800h, 16gb ram and an rtx 2060 there's no reason why I shouldn't be able to run this software properly

ikr

**Comment by w-okada** [comment]
Here is an experimentally updated version of ONNX Runtime for Apple Silicon.
I haven't been able to test it because I don't have the necessary environment, but it might improve the situation.

https://huggingface.co/wok000/vcclient000/resolve/main/MMVCServerSIO_mac_onnxcpu-nocuda_v.1.5.3.14_onnx15_1.zip

**Comment by brawlofnight** [comment]
> Here is an experimentally updated version of ONNX Runtime for Apple Silicon. I haven't been able to test it because I don't have the necessary environment, but it might improve the situation.
> 
> https://huggingface.co/wok000/vcclient000/resolve/main/MMVCServerSIO_mac_onnxcpu-nocuda_v.1.5.3.14_onnx15_1.zip

omg tysm it works a lot better just still a little choppy and laggy but so much better THANKS!

**Comment by BienGudBoy** [comment]
> > Here is an experimentally updated version of ONNX Runtime for Apple Silicon. I haven't been able to test it because I don't have the necessary environment, but it might improve the situation.
> > https://huggingface.co/wok000/vcclient000/resolve/main/MMVCServerSIO_mac_onnxcpu-nocuda_v.1.5.3.14_onnx15_1.zip
> 
> omg tysm it works a lot better just still a little choppy and laggy but so much better THANKS!

@brawlofnight How's your config? I'm trying on my M1 Air but the audio is still very choppy and unusable...

**Comment by brawlofnight** [comment]
> > > Here is an experimentally updated version of ONNX Runtime for Apple Silicon. I haven't been able to test it because I don't have the necessary environment, but it might improve the situation.
> > > https://huggingface.co/wok000/vcclient000/resolve/main/MMVCServerSIO_mac_onnxcpu-nocuda_v.1.5.3.14_onnx15_1.zip
> > 
> > 
> > omg tysm it works a lot better just still a little choppy and laggy but so much better THANKS!
> 
> @brawlofnight How's your config? I'm trying on my M1 Air but the audio is still very choppy and unusable...

its still pretty choppy and only really made it less laggier by like 10%. I don't know what a config is so ill just send a screenshot. https://imgur.com/a/fuZCPcv

**Comment by brawlofnight** [comment]
> Here is an experimentally updated version of ONNX Runtime for Apple Silicon. I haven't been able to test it because I don't have the necessary environment, but it might improve the situation.
> 
> https://huggingface.co/wok000/vcclient000/resolve/main/MMVCServerSIO_mac_onnxcpu-nocuda_v.1.5.3.14_onnx15_1.zip

when I said it was less laggier I meant that instead of random whispering 5 minutes after talking, it just made random whispering sounds after like 20 seconds. Even though, it is still very "unusable".
